Makefile

```diff
@@ -1,8 +1,13 @@
 SHELL = /bin/bash
 COMMIT = $(shell git rev-parse --short HEAD)$(shell [[ $$(git status --porcelain) = "" ]] || echo -dirty)
 ARO_IMAGE ?= ${RP_IMAGE_ACR}.azurecr.io/aro:$(COMMIT)
+OS=$(shell uname)
 
-export CGO_CFLAGS=-Dgpgme_off_t=off_t
+ifndef CGO_CFLAGS
+ifneq ("${OS}", "Darwin")
```

Comment:
It should probably be something like this

```Makefile
UNAME_S := $(shell uname -s)
ifndef CGO_CFLAGS
ifneq ("$(shell uname -s)", "Darwin")
	CGO_CFLAGS=-Dgpgme_off_t=off_t;
endif
endif
```

But I don't like it. See https://github.com/Azure/ARO-RP/pull/1082#issuecomment-712901017